Directory.Packages.props

```diff
@@ -5,45 +5,45 @@
     <AspireVersion>9.2.0</AspireVersion>

```

Comment:
should this be removed then if we arent using it

Reply:
Why did you stop using it in the first place?